lib/rake/ci.rake

```
@@ -93,6 +93,8 @@ namespace :ci do
       next
     end
 
+    check_for_new_file_changes
```

move this check to run before UI tests, so that we fail faster when there is a problem with a migration